public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -24,27 +24,41 @@ export interface Props {
   header?: string;
 }
 
+//string, unknown, string, boolean
+interface DisplayValue {
+  name: string;
+  value: unknown;
+  valueString: string;
+  highlight: boolean;
+}
+
 export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode, header = undefined }: Props) => {
   const styles = useStyles2(getStyles);
 
   if (!data || rowIndex == null) {
     return null;
   }
-
+  const fields = data.fields.map((f, idx) => {
+    return { ...f, highlight: idx === columnIndex };
```

Comment:
maybe we should name added property to field something a bit more descriptive?

maybe something like `isFieldHovered` - along those lines of the what vs. an effect of the property down the line?